lib/Models/GtfsCatalogItem.ts

```diff
@@ -258,7 +259,11 @@ export default class GtfsCatalogItem extends AsyncMappableMixin(
 
   @computed
   get mapItems(): DataSource[] {
-    return [this.dataSource];
+    if (isDefined(this.dataSource)) {
```

Comment:
Do we need the check here since the type of `get dataSource` guarantees that it is always a `DataSource`?

Reply:
Yeah I guess not, was just basing it off the other catalog items whoops!